core/pom.xml

```diff
@@ -220,6 +220,21 @@
       <version>3.2.10</version>
     </dependency>
     <dependency>
+      <groupId>com.fasterxml.jackson.module</groupId>
+      <artifactId>jackson-module-scala_2.10</artifactId>
+      <version>2.3.1</version>
+    </dependency>
+    <dependency>
+      <groupId>com.sun.jersey</groupId>
+      <artifactId>jersey-server</artifactId>
+      <version>1.9</version>
+    </dependency>
+    <dependency>
+      <groupId>org.glassfish.jersey.media</groupId>
+      <artifactId>jersey-media-json-jackson</artifactId>
```

Comment:
Could you double-check that pulling this is not causing different com.fasterxml.jackson libraries to be mixed up? I checked the pom for this artifact and it seems to depend on a different version of jackson than the one Spark pulls.

Also, I wonder if it wouldn't be better to do like Yarn and avoid pulling this dependency (which is from a different Jersey version). See:
https://github.com/apache/hadoop/blob/branch-2/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-common/src/main/java/org/apache/hadoop/yarn/webapp/YarnJacksonJaxbJsonProvider.java